tests/playTests.sh

```diff
@@ -390,6 +390,18 @@ println "\n===>  recursive mode test "
 # combination of -r with empty list of input file
 zstd -c -r < tmp > tmp.zst
 
+# combination of -r with empty folder
+mkdir -p tmpEmptyDir
+zstd -r tmpEmptyDir 2>tmplog2
+if [ grep "aborting" tmplog2 ]; then
```

Comment:
Seems this construction is still problematic for `sh` compatibility.

I'm no `sh` expert, and this specific construction is not used in our test script, 
but I can recommend an existing idiom that has worked well so far : 

```
zstd -r tmpEmptydir 2>&1 | grep "aborting" && die "Should not abort on empty directory"
```

This would be equivalent, but you could go for something even simpler : 

```
zstd -r tmpEmptydir 
```

this second test only relies on return code being successful, which, I believe, is the goal of the test ?

Reply:
thanks! my initial thought was to go with just zstd -r tmpEmptydir  but that I thought that test should be verbose. Changed to that. thanks!

Reply:
btw, "if then" sh syntax allows cleaning tmp dirs/files before exiting.